examples/somersaultecu.py

```diff
@@ -1008,10 +1003,45 @@ class SomersaultSID(IntEnum):
                     sdgs=[],
                 ),
             ]),
-            byte_size=None,
         ),
 }
 
+somersault_structures = {
```

Comment:
this is because requests and responses are no longer basic structures, so they cannot be referenced by STRUCT-REF anymore. (IOW, the somersault PDX file that is currently shipped with `main` is invalid...)